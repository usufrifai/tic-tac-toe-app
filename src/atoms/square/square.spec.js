import React from 'react';
import Square from './index';
import renderer from 'react-test-renderer';

describe('Square', () => {
    it('should render  Square Component', () => {
        const component = renderer.create(
            <Square 
               onClick={jest.fn()}
            />
          );
          let tree = component.toJSON();
          expect(tree).toMatchSnapshot();
    });

    it('should render  Square Component with O', () => {
        const component = renderer.create(
            <Square 
               value="O"
               onClick={jest.fn()}
            />
          );
          let tree = component.toJSON();
          expect(tree).toMatchSnapshot();
    });
  });