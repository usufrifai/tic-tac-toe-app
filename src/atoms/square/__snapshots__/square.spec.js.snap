// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Square should render  Square Component 1`] = `
<div
  className="squares"
  onClick={[MockFunction]}
>
  <span>
     
     
  </span>
</div>
`;

exports[`Square should render  Square Component with O 1`] = `
<div
  className="squares O"
  onClick={[MockFunction]}
>
  <span>
     
    O
     
  </span>
</div>
`;
